test-results/failing/GFM/HTML blocks.md

```diff
@@ -498,7 +498,7 @@ Should give output:
 But instead was:
 
 ````````````html
-ERROR Problem at row 1 Problem at row 2 Expecting symbol
+
```

Comment:
It's interesting that the results here are different from the other pull request: https://github.com/dillonkearns/elm-markdown/pull/84/files#diff-2f698e00f67e215049365798f048aefa26ec437849ee30ff6afb9341ef67b79fR501.